### ISSUE TITLE
Espec no longer passes mix options through

### ISSUE THREAD

**Issue by joffotron** [antonmi/espec#221]
Hi!

We were running espec like this:

```
mix espec --no-start --trace
```

Where it looks like the `--no-start` option was being passed through to mix, and thus our application would not try and start itself while the tests were running

Since version 1.4.2, now we get
```
** (Mix) Could not invoke task "espec": 1 error found!
--no-start : Unknown option
```

Is there a different way we should be invoking the `espec` task such that mix doesn't try and start our app?

**Comment by antonmi** [comment]
Hi, @joffotron !
I'm not sure, but it seems that this [commit] (https://github.com/antonmi/espec/commit/26bea41ede33e9bf6ec745ba3704d8e0670772e4) seems to broke options. 
I replaced `OptionParser.parse` with `parse!`.

**Comment by antonmi** [comment]
Remember, PR's are welcome!)

**Comment by antonmi** [comment]
Hi!
I released 1.4.5 with the fix!
